feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-overview-normalizer.ts

```diff
@@ -43,8 +43,7 @@ export class OccConfiguratorVariantOverviewNormalizer
     const characteristicValues: OccConfigurator.CharacteristicOverview[] =
```

Comment:
Unit tests fail for OccConfiguratorVariantNormalizer

Reply:
The tests have been fixed